examples/advanced/extensions/parallel_plot/parallel_plot.py

```diff
@@ -1,9 +1,8 @@
 import numpy as np
 
 from bokeh.layouts import column
-from bokeh.models import (BasicTickFormatter, ColumnDataSource,
-                          CustomJSTickFormatter, Div, FixedTicker,
-                          LinearAxis, LinearColorMapper, MultiLine, Range1d)
+from bokeh.models import (BasicTickFormatter, ColumnDataSource, CustomJSTickFormatter,
+                          Div, FixedTicker, LinearAxis, LinearColorMapper, Range1d)
```

Comment:
While working on some examples I also fixed some PEP-Warnings and had to rearange some lines for the code style checks. This is why some files have "many" changes.

Reply:
It would nice if we could enable the same style of imports, that we use under `src/`, for when we need to import a lot of models, etc.

Reply:
If I change the style of the imports the pre-push-hook `test_isort` fails.

We could add [`-m=3`](https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes.html) to the line below

https://github.com/bokeh/bokeh/blob/df57080f22b39816deee84d5e835f752d1713925/tests/codebase/test_isort.py#L58

but this woul mean, we have to change all import in all files. This would be worth a new issue. Is it okey for you if I open a seperate PR, to make this change to all import in this project?

Reply:
The import style under `examples` was made different from that under `src` on purpose. We can discuss whether that should be the case or not any longer, but we should not take any action regarding that in this PR